client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab/index.ts

```diff
@@ -124,96 +154,42 @@ const usePersonalInfoTab = (parameters: usePersonalInfoTabParameters): usePerson
-        axios.get('/personalDetails/educationSubOccupations?city=' + city).then((res: any) => {
-            if (res && res.data && res.data.data) {
-                educationSubOccupationsLogger.info('got results from the server', Severity.LOW);
-                setSubOccupations(res.data.data.allSubOccupations.nodes.map((node: any) => {
-                    return {
-                        id: node.id,
-                        subOccupation: node.displayName,
-                        street: node.street
-                    }
-                }));
-            } else {
-                educationSubOccupationsLogger.warn('got status 200 but got invalid outcome', Severity.HIGH);
-            }
-        });
-    }
-
-    const getStreetsByCity = (cityId: string) => {
-        const streetsByCityLogger = logger.setup('Getting streets of city');
-        streetsByCityLogger.info(`launching request to server with parameter ${cityId}`, Severity.LOW);
-        axios.get('/addressDetails/city/' + cityId + '/streets').then((res: any) => {
-            if (res && res.data) {
-                streetsByCityLogger.info('got data from the server', Severity.LOW);
-                setStreets(res.data.map((node: any) => (
-                    {
-                        displayName: node.displayName,
-                        id: node.id
-                    }
-                )));
-            } else {
-                streetsByCityLogger.warn('got status 200 but wrong data', Severity.HIGH);
-            }
-        });
-    }
+    const clearSubOccupations = () => setSubOccupations([]);
 
     const savePersonalData = (personalInfoData: PersonalInfoDbData, data: { [x: string]: any }, id: number) => {
         const savePersonalDataLogger = logger.setup('Saving personal details tab');
         savePersonalDataLogger.info('launching the server request', Severity.LOW);
         setIsLoading(true);
-        axios.post('/personalDetails/updatePersonalDetails',
-            {
-                id: investigatedPatientId,
-                personalInfoData,
-            })
-            .then(() => {
-                const isInvestigationNew = investigationStatus.mainStatus === InvestigationMainStatusCodes.NEW;
-                savePersonalDataLogger.info(
-                    `saved personal details successfully${isInvestigationNew ? ' and updating status to "in progress"' : ''}`,
-                    Severity.LOW
-                );
-            })
-            .catch((error) => {
-                savePersonalDataLogger.error(`got error from server: ${error}`, Severity.HIGH);
-                complexityErrorAlert(error);
-            })
-            .finally(() => {
-                setIsLoading(false);
-                personalInfoValidationSchema.isValid(data).then(valid => {
-                    setFormState(epidemiologyNumber, id, valid);
-                })
+        axios.post('/personalDetails/updatePersonalDetails', {
+            id: investigatedPatientId,
```

Comment:
isn't it just epidemiologyNumber? isn't it supposed to be send already as a header?

Reply:
no,  it is the id in table investigated_patient